test/benchmarks_suite.go

```diff
@@ -0,0 +1,115 @@
+package test
```

Comment:
We have several benchmarks now. And they are packaged into a suite that can be called on any store implementation.